lib/chat_api/conversations.ex

```diff
@@ -23,6 +23,19 @@ defmodule ChatApi.Conversations do
     |> Repo.all()
   end
 
+  @spec list_conversations_by_account_paginated(binary(), map()) :: Paginator.Page.t()
+  def list_conversations_by_account_paginated(account_id, filters \\ %{}) do
+    Conversation
+    |> where(account_id: ^account_id)
+    |> where(^filter_where(filters))
+    |> where([c], is_nil(c.archived_at))
+    |> order_by(desc: :last_activity_at, desc: :id)
```

Comment:
I'm curious, why is the sort different than both the v1 above for the v2 below? I assume this is what prevents you from duplicating the code. Is this a sorting specific thing?

Reply:
Yeah this PR is a continuation of this other PR https://github.com/papercups-io/papercups/pull/634
We decided to create a `last_activity_at` field in the `conversations` table so that we don't have a lateral join to the `message` field to sort which conversation has the most recent message, which the method below uses